docs/pages/database-access/enroll-google-cloud-databases/spanner.mdx

```diff
@@ -0,0 +1,217 @@
+---
+title: Database Access with Cloud Spanner
+description: How to configure Teleport database access with GCP's Cloud Spanner.
+---
+
+(!docs/pages/includes/database-access/db-introduction.mdx dbType="Cloud Spanner" dbConfigure="with a service account"!)
+
+## How it works
+
+(!docs/pages/includes/database-access/how-it-works/iam.mdx db="Spanner" cloud="Google Cloud"!)
+
+<Tabs>
+
+<TabItem label="Self-Hosted">
+![Teleport Cloud Spanner Self-Hosted](../../../img/database-access/guides/spanner_selfhosted.png)
+</TabItem>
+
+<TabItem label="Cloud-Hosted">
+![Teleport Cloud Spanner Cloud-Hosted](../../../img/database-access/guides/spanner_cloud.png)
+</TabItem>
+
+</Tabs>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- Google Cloud account
+- Google Cloud Spanner database.
+- A host where you will run the Teleport Database Service, e.g. a Compute Engine
+  instance.
+- (!docs/pages/includes/tctl.mdx!)
+
+## Step 1/8. Create a service account for the Teleport Database Service
+
+(!docs/pages/includes/database-access/cloudsql-create-service-account-for-db-service.mdx !)
+
+Ignore the optional steps - click "Done".
+We will grant permissions to this service account later in this guide.
+
+### Create a key for the service account
+
+(!docs/pages/includes/database-access/cloudsql_service_account_key.mdx!)
+
+## Step 2/8. Create a service account for a database user
+
+When a user connects to Spanner via Teleport, they will specify a service
+account name as the "database user" and the Teleport Database service will
+impersonate that service account to authenticate to GCP.
+
+You can create multiple service accounts to provide different access levels to
+Teleport users, but for this guide we will just create one. 
+
+(!docs/pages/includes/database-access/cloudsql_create_db_user_account.mdx!)
+
+Ignore the optional steps - click "Done".
+Rather than granting access at the project level, we will grant this service
+account permissions to just the databases it will be used to access.
+
+### Grant permissions
+
+Navigate to the
+[Spanner instance overview page](https://console.cloud.google.com/spanner/instances)
+and check the box of your Spanner instance, then click "Permissions".
+
+![Open Cloud Spanner Instance Permissions](../../../img/database-access/guides/cloudsql/select-instance@2x.png)
+
+In the permissions blade, click "Add Principal" then add the "teleport" service
```

Comment:
I agree, I use it only because that's generally the example db user we use.

I'll update it to "spanner-user".
I'll also update the other GCP guides to use a more specific name